utils/mk-squash-iso.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+help() {
+	cat <<EOF
+mk-squash-iso.sh [options]
+	Turns an sabotage rootfs into an bootable iso.
+	This requires an working isolinux installation on <rootfs>/boot.
+
+	-o <file>
+		Set the output file. Existing files will be overwritten
+		without asking. The file will be an iso image.
+
+	-l <label>
+		Define a custom disk label.
+
+	-r <dir>
+		Set the source rootfs. The rootfs itself will not be changed.
+
+	-t
+		Exclude tarballs directory.
```

Comment:
I decided against adding an custom selector function since its a big hazzle to implement.